game_python/GameLogic.py

```diff
@@ -84,21 +84,28 @@ def make_game_state(self, state):
         Args:
             state (str): The game state received from the server.
         """
+        state = state.replace("\x00", "")
         winner = int(state[0])
         number_of_players = int(state[1])
         who_to_move = int(state[2])
         message = int(state[3])
         self.gameState = GameState(
-            winner, number_of_players, who_to_move, message, [Player() for _ in range(number_of_players)])
+            winner,
+            number_of_players,
+            who_to_move,
+            message,
+            [Player() for _ in range(number_of_players)],
+        )
         for i in range(number_of_players):
-            index = 4 + i*20
-            self.gameState.players[i].name = state[index:index+14]
+            index = 4 + i * 20
```

Comment:
If operators with different priorities are used, consider adding whitespace around the operators with the lowest priority(ies). Use your own judgment; however, never use more than one space, and always have the same amount of whitespace on both sides of a binary operator:
```
# Correct:
i = i + 1
submitted += 1
x = x*2 - 1
hypot2 = x*x + y*y
c = (a+b) * (a-b)
```
```
# Wrong:
i=i+1
submitted +=1
x = x * 2 - 1
hypot2 = x * x + y * y
c = (a + b) * (a - b)
```